store/sqlstore/channel_store.go

```diff
@@ -1392,7 +1395,7 @@ func (s SqlChannelStore) SaveMultipleMembers(members []*model.ChannelMember) ([]
 	}
 	defer finalizeTransaction(transaction)
 
```

Comment:
As `saveMultipleMembersT` is  not using the `transaction ` the transaction in `SaveMultipleMembers`  is now redundant. What do you think?

Reply:
Sure, we can do that. But I'd prefer to keep the unparam changes separate from this transaction removal.